XcodeServerSDK.xcodeproj/xcshareddata/xcschemes/XcodeServerSDK - OS X.xcscheme

```diff
@@ -29,7 +29,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "3A7B48C41B2A5AC40077ABEA"
-               BuildableName = "XcodeServerSDK - OS X Tests.xctest"
+               BuildableName = "XcodeServerSDKTest.xctest"
```

Comment:
I'm getting my Xcode also trying to rename the buildable names of the test targets. Anyone have a clue why? @esttorhe @cojoj